apps/arweave/include/ar.hrl

```diff
@@ -310,6 +310,23 @@
 %% Maximum size of a single data chunk, in bytes.
 -define(DATA_CHUNK_SIZE, (256 * 1024)).
 
+%% The maximum allowed packing difficulty.
+-define(MAX_PACKING_DIFFICULTY, 32).
+
+%% The number of sub-chunks in a compositely packed chunk with the packing difficulty
+%% between 1 and 32 incl. The composite packing with the packing difficulty 1 matches
+%% approximately the non-composite 2.6 packing in terms of computational costs.
+-define(PACKING_DIFFICULTY_ONE_SUB_CHUNK_COUNT, 32).
+
+%% The size of a unit sub-chunk in the compositely packed chunk with the packing difficulty
+%% between 1 and 32 incl.
+-define(PACKING_DIFFICULTY_ONE_SUB_CHUNK_SIZE,
+		(?DATA_CHUNK_SIZE div ?PACKING_DIFFICULTY_ONE_SUB_CHUNK_COUNT)).
+
+%% The number of RandomX rounds used for a single iteration of packing of a single sub-chunk
+%% during the composite packing with the packing difficulty between 1 and 32 incl.
+-define(PACKING_DIFFICULTY_ONE_ROUND_COUNT, 10).
```

Comment:
Why do we use 10 here, but down [here](https://github.com/ArweaveTeam/arweave/pull/590/files#diff-f39d3764650cee3f08a44af8852d72bb770d091e7cdfb3c7539efa092b030569R19-R25) we use 8?

Reply:
The latter (8) is not used, I removed it.